src/mappings/constants.ts

```diff
@@ -1 +1,12 @@
 export const PREFIX = "poktroll";
+
+export const StakeStatus = {
+  Staked: 0,
+  Unstaking: 1,
+  Unstaked: 2,
+} as const
+
+export const TxStatus = {
+  Success: 0,
+  Error: 1,
+} as const
```

Comment:
~~Why not use [enums](https://www.typescriptlang.org/docs/handbook/enums.html)?~~ See comments in schema.graphql.

Reply:
We are going to use typescript enum